binder/symedoctests.yml

```diff
@@ -0,0 +1,15 @@
+name: symengine_conda_doctests
```

Comment:
Why do we need this file?

Reply:
We only really need `papermill` and `jupytext`, I'll just `conda install` them in GHA; thanks

Reply:
Would it be better to add to the other environment @isuruf? I notice it has `notebook` which isn't a build dependency...